### PR TITLE
Take a riff off Asimov's first law of robotics

### PR DESCRIPTION
**Pull request by sdake**
This simply rewords some of Asimov's great thinking and applies to the values of a the Istio project.

Reference: https://en.wikipedia.org/wiki/Three_Laws_of_Robotics

Note the next revision uses many concepts of the first law. This revision takes a different approach.

Cheers
-steve

### CODE REVIEW

**Comment by linsun** [comment]
@sdake can you put a reference for Asimov?

File: VALUES.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 We want to make sure every member has a shared understanding of the goals and
 values we hold as a team:
 
+- Istio may not endanger a consuming organization's objectives or, through
```
Comment (linsun) [inline VALUES.md]: What is the consuming org here?  Is this vendors building solutions on top of istio or users?

Comment (sdake) [reply VALUES.md]: reworded.

**Comment by sdake** [comment]
> I think I like what you're trying to get at but I'm not sure we're quite there yet. Maybe something a little more prescriptive.
> 
> ' Design for the user not for yourself - Always consider what our users are trying to achieve with Istio. Many of them have a made a long term investment in the project and arbitrary changes can have major impacts on their plans'

Looks good. With your consent, I'll add you as coauthor and integrate the language.

Cheers
-steve

**Pull request merged**